src/System.Text.Json/src/System/Text/Json/ThrowHelper.Serialization.cs

```diff
@@ -28,6 +28,11 @@ public static NotSupportedException GetNotSupportedException_SerializationNotSup
             return new NotSupportedException(SR.Format(SR.SerializationNotSupportedCollectionType, propertyType));
         }
 
+        public static void ThrowInvalidOperationException_SerializerCycleDetected(int maxDepth)
+        {
+            throw new JsonException(SR.Format(SR.SerializerCycleDetected, maxDepth));
```

Comment:
Should we add the exception info that was being added previously? By calling `AddExceptionInformation`?

https://github.com/dotnet/corefx/blob/5079d5396fd0f847b1ed964d6329e89bc142df5c/src/System.Text.Json/src/System/Text/Json/ThrowHelper.Serialization.cs#L49-L55

Reply:
That was redundant since it is now added automatically for JsonException [here](https://github.com/dotnet/corefx/blob/1841042b99062de13dc80098cede9413be569238/src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.cs#L79).

Also the test verified it is added (basically the Path property is set).

Reply:
Should we remove it in other places too, then?